test/TensorFlowNET.UnitTest/NativeAPI/Eager/CApi.Eager.OpInferMixedTypeInputListAttrs.cs

```diff
@@ -44,15 +42,14 @@ static SafeContextHandle NewContext(SafeStatusHandle status)
                 //EXPECT_EQ(attr_found->second.list().type(1), tensorflow::DataType::DT_FLOAT);
                 //EXPECT_EQ(attr_found->second.list().type(2), tensorflow::DataType::DT_INT32);
 
-                int num_retvals = 1;
-                TFE_Execute(assertOp, retvals, ref num_retvals, status);
+                var retvals = new SafeTensorHandleHandle[1];
+                int num_retvals;
+                TFE_Execute(assertOp, retvals, out num_retvals, status);
                 EXPECT_EQ(TF_OK, TF_GetCode(status), TF_Message(status));
-            }
+                EXPECT_EQ(1, num_retvals);
 
-            TFE_DeleteTensorHandle(condition);
-            TFE_DeleteTensorHandle(t1);
-            TFE_DeleteTensorHandle(t2);
-            TFE_DeleteTensorHandle(retvals[0]);
```

Comment:
😨 I'm scared to think what this line was doing before

Reply:
Ah, it was hitting this:
https://github.com/tensorflow/tensorflow/blob/d05aa8735f35e6c6d43fdec4c63455912e318298/tensorflow/c/eager/c_api.cc#L940